internal/database/repository/partitions.go

```diff
@@ -54,19 +57,18 @@ func (s *PostgresRepository) UpsertPartitions(ctx context.Context, partitions []
 	return nil
 }
 
-func (s *PostgresRepository) GetAllPartitions(ctx context.Context) ([]*dao.PartitionInfo, error) {
+func (s *PostgresRepository) GetAllPartitions(ctx context.Context) ([]*model.PartitionInfo, error) {
```

Comment:
Should we be preparing for pagination parameters here? It seems unlikely we'd ever want to return everything to the front end all at once...

Reply:
Thanks, we also have several other APIs that need to be updated with pagination. This task is tracked in https://github.com/G-Research/yunikorn-history-server/issues/245